arangopipe/tests/test_data_generator/generate_model_data.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jul 12 07:25:18 2019
+
```

Comment:
Next time we should add the Apache 2 license, I will create a PR for the existing files

Reply:
sure. I will make sure I include the Apache 2 license in the data generators for hyper-parameters and exploratory data anlaysis.